### DISCUSSION TITLE
Thank you for making this plugin!

### DISCUSSION THREAD

**Discussion by ziontee113** [woosaaahh/sj.nvim#15] [General]
Hi @woosaaahh , I've just tried out sj.nvim and I'm having very good first impressions.

This plugin gives me the "very lightweight" feeling, and feels very natural to use :+1: 

Thank you so much for making this plugin. :heart:

**Comment by woosaaahh** [comment]
Wooow @ziontee113 is here !!!

Thank you so much for your comment ! I really appreciate it !

> This plugin gives me the "very lightweight" feeling, and feels very natural to use +1

Very cool, I'm glad you like it !
